src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -78,15 +78,74 @@
     'index': 'not_analyzed'
 }
 
+
 class ElasticsearchError(Exception):
+    """ Not operational errors. """
     pass
 
+
 class EmptySearchResultError(ElasticsearchError):
     pass
 
+
 class TooManyResultsError(ElasticsearchError):
     pass
 
+
+def setup(hosts):
+    """
+    Initialize Elasticsearch client and share it as the attribute _es_client in
+    the current module. An additional attribute _es_hosts is defined containing
+    the Elasticsearch hosts (expected types are: string, list or tuple).
+    """
+    module = sys.modules[__name__]
+    setattr(module, '_es_hosts', hosts)
+    setattr(module, '_es_client', Elasticsearch(**{'hosts': _es_hosts}))
```

Comment:
How about this?
```
_es_hosts = None
_es_client = None

def setup(hosts):
    global _es_hosts
    global _es_client
    # ...
    _es_hosts = hosts
    _es_client = Elasticsearch(**{'hosts': _es_hosts})
```

I think you only check for truthiness on _es_hosts & _es_client, and I think it's clearer to know that they're module attributes (and it keeps my linter from yowling).